src/subcommand/rna_main.cpp

```diff
@@ -250,6 +251,21 @@ int32_t main_rna(int32_t argc, char** argv) {
         return 1;       
     }
 
+    // taken from https://stackoverflow.com/a/64794991/
+    for (const auto & f : {std::cref(transcript_filenames), std::cref(intron_filenames)}) {
```

Comment:
This might be too structural to *need* to attribute, and "taken from" makes it sound like you've grabbed a whole snippet or block and not a technique. But it's also something I've never seen before, and the link is less useful for explaining what you mean than a comment and a good variable name.

I feel like the way I would write this wold be:
```suggestion
    for (const auto& filenames : {std::cref(transcript_filenames), std::cref(intron_filenames)}) {
        // For each collection of filenames (see https://stackoverflow.com/a/64794991/)
```